src/poetry_plugin_freeze/app.py

```diff
@@ -12,8 +12,9 @@
 
 from cleo.helpers import option
 from poetry.console.commands.command import Command
+import poetry.core
+from packaging.version import parse as parse_semver
```

Comment:
I think with a different import approach (in a comment below) we can avoid the reliance on version sniffing.

```suggestion
```